src/test/java/com/gmail/service/UserServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package com.gmail.service;
+
+import com.gmail.config.JavaConfig;
+import com.gmail.model.User;
+import junit.framework.TestCase;
+import org.junit.Before;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import javax.sql.DataSource;
+import java.time.LocalDate;
+import java.util.List;
+
+public class UserServiceImplTest extends TestCase {
```

Comment:
Please fix tests and run them. Are they all green?

Reply:
http://prntscr.com/nnm0cy

Reply:
There are no tests or you use junit 3?

Reply:
Tests are there

Reply:
I have to record video?

Reply:
Should I record a video?

Reply:
No.
1) But better not to extends from class TestCase but use annotation @Test.
2)What did you test in this testclass. From the test class name, it should be tests for the service layer. 
So why did you use spring jbdc templates here? Why not create seperate configuration for tests and use it. Instead of production data use some mocks or separate db. For example h2 database.